terraform-plan-comment/src/generate-outputs.js

```diff
@@ -90,6 +92,12 @@ const generateOutputs = async (workingDirectory, planFile, maxThreads, ignoredRe
     .then((output) => filterIgnored(output, ignoredResourcesRegexp))
     .then(sortModulePaths)
     .then((changed) => {
+      changed.forEach(({ plan }) => {
+        const planwWithChanges = `${plan}.changes`;
+        fs.copyFile(plan, planwWithChanges, () => {
```

Comment:
Maybe use copyFileSync to make sure they all succeed?

Reply:
I think no matter for the final result to use sync or async copy here.